src/elements/annotation-thread/AnnotationThread.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @flow
```

Comment:
We're trying to migrate box-ui-elements to typescript so new development should be written in typescript, if possible